src/main/java/seedu/application/logic/commands/EditCommand.java

```diff
@@ -24,17 +29,23 @@ public class EditCommand extends Command {
     public static final String COMMAND_WORD = "edit";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the job identified "
-            + "by the index number used in the displayed job list. "
-            + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
-            + "[" + PREFIX_ROLE + "ROLE] "
-            + "[" + PREFIX_COMPANY + "COMPANY] \n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_ROLE + "Software Engineer "
-            + PREFIX_COMPANY + "Google";
+        + "by the index number used in the displayed job list. "
+        + "Existing values will be overwritten by the input values.\n"
+        + "Parameters: INDEX (must be a positive integer) "
+        + "[" + PREFIX_ROLE + "ROLE] "
+        + "[" + PREFIX_COMPANY + "COMPANY] "
+        + "[" + PREFIX_DEADLINE + "DEADLINE] "
+        + "Example: " + COMMAND_WORD + " 1 "
+        + PREFIX_ROLE + "Software Engineer "
+        + PREFIX_COMPANY + "Google"
+        + PREFIX_DEADLINE + "Dec 31 2023 1200";
 
     public static final String MESSAGE_EDIT_JOB_SUCCESS = "Edited Job: %1$s";
-    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided. \n"
+        + "c/ for Company\n"
```

Comment:
You might want to use `PREFIX_ROLE`, `PREFIX_COMPANY` etc to replace r/, c/ here to keep it consistent.